<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Porn Fetch 1.0 (CC)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupbox_urls">
     <property name="title">
      <string>Download</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_url">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
          </property>
          <property name="text">
           <string>URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineedit_url">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_start">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #5468ff;
    border: none;
    border-radius: 6px;
    color: #fff;
    font-family: &quot;JetBrains Mono&quot;, monospace;
    font-size: 15px;
    padding: 0px 16px 0px 16px;
    height: 24px;
}

QPushButton:hover {
    background-color: #3c4fe0;
}

QPushButton:pressed {
    background-color: #3c4fe0;
}
</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_url_file">
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
          </property>
          <property name="text">
           <string>File with URLs:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineedit_url_file">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_start_file">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #5468ff;
    border: none;
    border-radius: 6px;
    color: #fff;
    font-family: &quot;JetBrains Mono&quot;, monospace;
    font-size: 15px;
    padding: 0px 16px 0px 16px;
    height: 24px;
}

QPushButton:hover {
    background-color: #3c4fe0;
}

QPushButton:pressed {
    background-color: #3c4fe0;
}
</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupbox_settings">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupbox_useless">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupbox_quality">
           <property name="styleSheet">
            <string notr="true">color: rgb(122, 0, 255)</string>
           </property>
           <property name="title">
            <string>Quality</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radio_highest">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton {
	color: rgb(0, 255, 152)}


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}
</string>
              </property>
              <property name="text">
               <string>Highest</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radio_middle">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton {
	color: rgb(255, 171, 0)}


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}

</string>
              </property>
              <property name="text">
               <string>Middle</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radio_lowest">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton {
	color: rgb(255, 0, 0)}


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}
</string>
              </property>
              <property name="text">
               <string>Lowest</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_output">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255)</string>
             </property>
             <property name="text">
              <string>Output Path:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineedit_output">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupbox_settings_ui">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 176, 0)</string>
        </property>
        <property name="title">
         <string>UI</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radio_native">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: rgb(0, 255, 250) }


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}
</string>
           </property>
           <property name="text">
            <string>Native</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radio_dark">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: rgb(81, 0, 255)}


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}
</string>
           </property>
           <property name="text">
            <string>Dark mode</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radio_white">
           <property name="cursor">
            <cursorShape>ForbiddenCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton {
	color: rgb(255, 255, 255)}


QRadioButton::indicator::unchecked {
	border: 1px solid white;
	border-radius: 5px;
}


QRadioButton::indicator:checked {
    border : 4px solid;
	border-color: black;
	border-radius: 6px;
	background-color: rgb(0, 255, 183);

}
</string>
           </property>
           <property name="text">
            <string>White mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QProgressBar" name="progressbar_download">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    background-color: #F0F0F0; /* Hellgrauer Hintergrund */
	text-align: center;
	color: rgb(230, 97, 0);
	border: color grey;
	border-width: 6;
	border-radius: 12px;
	color: black;
	
}

QProgressBar::chunk {
    background-color: rgb(26, 95, 180); /* Grün als Vordergrundfarbe */
}cc</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupbox_metadata">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="title">
      <string>Metadata</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="4" column="0">
       <widget class="QLabel" name="label_duration">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_author">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Author: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineedit_author">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineedit_duration">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineedit_title">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineedit_hotspots">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_hotspots">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Hotspots:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_date">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineedit_views">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineedit_date">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #757575;
    border-radius: 5px;
    padding: 0 8px;
    background: rgb(94, 92, 100);  /* setzt den Hintergrund auf Schwarz */
    color: #FFFFFF;  /* setzt die Textfarbe auf Weiß */
    font-size: 16px;
    height: 20px;
}

QLineEdit:focus {
    border-color: #4a90d9;
}

QLineEdit:disabled {
    background: #444444;  /* setzt den Hintergrund auf ein dunkles Grau, wenn das QLineEdit deaktiviert ist */
    color: #aaaaaa;  /* setzt die Textfarbe auf ein helles Grau, wenn das QLineEdit deaktiviert ist */
    border-color: #aaaaaa;
}</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_views">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: white;
    font-size: 16px;
	margin-right: 28px;
}

QLabel#Title {
    color: #212121;
    font-size: 24px;
    font-weight: bold;
}

QLabel#Subtitle {
    color: #757575;
    font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Views:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="button_get_metadata">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #5468ff;
    border: none;
    border-radius: 6px;
    color: #fff;
    font-family: &quot;JetBrains Mono&quot;, monospace;
    font-size: 15px;
    padding: 0px 16px 0px 16px;
    height: 24px;
}

QPushButton:hover {
    background-color: #3c4fe0;
}

QPushButton:pressed {
    background-color: #3c4fe0;
}
</string>
        </property>
        <property name="text">
         <string>Get metadata</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="credits_text">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0)</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Credits:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;API: PHUB by Egsagon.  This project would not be possible without it.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Author: EchterAlsFake&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;License: Creative Commons&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Plugins: Tabnine, Material Theme Icons&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Libraries: colorama, tqdm, PySide6&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Graphical User Interface was created with Qt - PySide6&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Version: 1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
